.circleci/config.yml

```diff
@@ -25,18 +38,7 @@ workflows:
           context: plugin-ci
           requires:
             - plugin-ci/lint
-            - plugin-ci/test
-            - plugin-ci/build
-      - plugin-ci/deploy-release:
```

Comment:
is the deploy-release job obsolete? Is deploy-release-github the official job that is used for releases?

Reply:
I is obsolete. There is a ticket to remove it from every circleci config: https://mattermost.atlassian.net/browse/MM-23138